### ISSUE TITLE
Via stuck on authorize device page after "authorizing device".

### ISSUE THREAD

**Issue by jimklo** [the-via/releases#221]
On Ubuntu 22.04. Tried both Vivaldi and Chrome browsers same result.

1. Navigate to https://usevia.app/#/ 
2. Prompted to select at WebHID device, I select my keyboard (KBDFans KBD67mkiiRGB v2) and click connect
3. Nothing else happens.

Keyboard used is KBDFans KBD67lite

**Comment by JackTLi** [comment]
Same problem with Keychron Q10.
Seeing this 404 - not sure if related.

![image](https://user-images.githubusercontent.com/9454024/206960644-d0d54d11-5709-422f-82d9-e307e20fbbf1.png)

**Comment by cyrenasi** [comment]
> Same problem with Keychron Q10. Seeing a 404 on ` https://usevia.app/definitions/v2/875823521.json` , not sure if related

I got the same problem with my Keychron V3 and solved by following the suggestion from @lubaxe on issue #211 . 
Toggle the "Use V3 Definitions" to OFF.

**Comment by wilba** [comment]
See https://github.com/the-via/app/issues/91

**Comment by appetrosyan** [comment]
Same problem with a fresh k13 Pro. Would not have bought, had  I known it's going to be such a mess.

**Comment by oscaracena** [comment]
And again, same problem with a NuPhy Air75 v2. There is a [JSON definition](https://nuphy.com/pages/json-files-for-nuphy-keyboards), which I loaded, but when I select the device, it does not change anything. :/

**Comment by Raymondoor** [comment]
> And again, same problem with a NuPhy Air75 v2. There is a [JSON definition](https://nuphy.com/pages/json-files-for-nuphy-keyboards), which I loaded, but when I select the device, it does not change anything. :/

you have to update the firmware I guess. I had the same issue, but now works. 
[Here is the firmware](https://nuphy.com/pages/qmk-firmwares#s-72cf4613-f599-48b5-aeb2-e57d5416425b)

**Comment by oscaracena** [comment]
Yes! That was the problem. I've updated the firmware and now it works flawlessly. Thank you very much! 😃

**Comment by appetrosyan** [comment]
> I've updated the firmware and now it works flawlessly. Thank you very much! 😃

Then the problem is that the via app doesn't tell you that you need to update the firmware....

**Comment by D3RS56** [comment]
Hello, I am having this same issue with the Shelby65.

**Comment by obear** [comment]
... and I have the the same problem with the keychron Q2 max (with updated firmware)

**Comment by ThreeM-MMM** [comment]
Same problem here. Its not firmware related, in Microsoft Edge, via works.

**Comment by gouglov** [comment]
same problem here :
Linux Mint, using chromium. Keychron Launcher works fine, but not VIA

**Comment by IntelligentFunction** [comment]
Yup same problem on NixOS using Lemokey P1 pro. I tried the standalone app too, and that just won't let you click the confirm button even.
